### ISSUE TITLE
Lyrics3v2 Tag Support

### ISSUE THREAD

**Issue by MariusBinary** [mono/taglib-sharp#100]
Why this library does not support the Lyrics3v2 Tag?

**Comment by decriptor** [comment]
taglib-sharp is put together largely by people that have added the bits and pieces that they needed for their stuff.

If you have time and feel up to it we'd be happy if someone submitted support for Lyrics3v2 tags

**Comment by MariusBinary** [comment]
Yes, Lyrics3v2 tag is a type of syncronized lyrics but it is different from ID3v2 syncronized lyrics.
